### PR TITLE
include support for ActiveRecord 6.1

### PR DESCRIPTION
**Pull request by nehresma**
Addresses #178.  Specs pass locally but haven't tested with Travis.

### CODE REVIEW

File: gemfiles/mysql2/6-1.gemfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  platforms :ruby, :mswin, :mingw do
+    gem "mysql2"
+  end
+
+  gem "coveralls", require: false
+  gem "rspec", ">= 2.11"
+  gem "simplecov", require: false
+
+  gem "activerecord", "~> 6.1.0"
```
Comment (albus522) [inline gemfiles/mysql2/6-1.gemfile:20]: This does not resolve until 6.1.0 is out. This does not match release candidates.

Comment (nehresma) [reply gemfiles/mysql2/6-1.gemfile:20]: Yes, you are correct.  I perhaps was jumping the gun with the PR, anticipating Rails 6.1 in the next few days and hoping to be able to upgrade when it arrives.

File: delayed_job_active_record.gemspec [hunk 2]
```diff
@@ -12,5 +12,5 @@ Gem::Specification.new do |spec|
   spec.name           = "delayed_job_active_record"
   spec.require_paths  = ["lib"]
   spec.summary        = "ActiveRecord backend for DelayedJob"
-  spec.version        = "4.1.4"
+  spec.version        = "4.1.5"
```
Comment (albus522) [inline delayed_job_active_record.gemspec:15]: Assuming you make additional open source contributions in the future, never include a gem version bump in your PR.

**Comment by albus522** [comment]
Thank you for the contribution, but there is enough that I need to tweak here that I am going to start from a clean master copy. I will hopefully have a release out before the end of the day.

**Pull request closed without merging**